R/db-io.R

```diff
@@ -142,14 +152,14 @@ dbi_quote.ident <- function(x, con) DBI::dbQuoteIdentifier(con, as.character(x))
 dbi_quote.character <- function(x, con) DBI::dbQuoteString(con, x)
 dbi_quote.sql <- function(x, con) DBI::SQL(as.character(x)) # nocov
 
-create_indexes <- function(con, table, indexes = NULL, unique = FALSE, ...) {
```

Comment:
I _think_ you should leave this as is since `.call` is only available in some methods.